txcache/transactionsTracker.go

```diff
@@ -119,6 +117,21 @@ func (txTracker *transactionsTracker) IsTransactionTracked(transaction *WrappedT
 	return true
 }
 
+func (txTracker *transactionsTracker) GetBulkOfUntrackedTransactions(transactions []*WrappedTransaction) [][]byte {
+	untrackedTransactions := make([][]byte, 0, len(transactions))
```

Comment:
```suggestion
	untrackedTransactions := make([][]byte, 0)
```

due to the continue statement from L124 and the condition from L127, this might be initialized with more space than needed